### PR TITLE
Add timer types

### PR DESCRIPTION
**Pull request by caeleel**
Adds typings for the new timer plugin API.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -8,12 +8,15 @@ declare global {
     [key: string]: string
   }
 
+  type EventType = "selectionchange" | "currentpagechange" | "close" | "timerstart" | "timerstop" | "timerpause" | "timerresume" | "timeradjust" | "timerdone"
```
Comment (ymichael) [inline index.d.ts:11]: tyty

File: index.d.ts [hunk 2]
```diff
@@ -8,12 +8,15 @@ declare global {
...
+
   interface PluginAPI {
     readonly apiVersion: "1.0.0"
     readonly command: string
 
     readonly fileKey: string | undefined
 
+    readonly timer: TimerAPI
```
Comment (ymichael) [inline index.d.ts:19]: should this be timer?

**Review by ymichael — APPROVED** [review]

File: index.d.ts (hunk 2 above)
Comment (caeleel) [reply index.d.ts:19]: erm, can you elaborate?

Comment (ymichael) [reply index.d.ts:19]: omg re-reading my comment 🤦🏻 

I meant to say... should this be `timer?` ?

Comment (caeleel) [reply index.d.ts:19]: ohhh haha thank you! yes

**Comment by minigolf2000** [comment]
@caeleel Hmm I had think this PR deleted the `"run"` event is that intentional?

**Pull request merged**